source/localizable/object-model/computed-properties-and-aggregate-data.md

```diff
@@ -46,7 +46,7 @@ export default Ember.Component.extend({
 In both of the examples above, `incomplete` is an array containing the single incomplete todo:
 
 ```javascript
-import TodosComponent from 'app/components/todos';
+import TodosComponent from 'app/components/todos-list';
```

Comment:
Seems like it would be cleanest to change this line (and the lines beneath) to `TodoListComponent` as well ...